spec/integration/sequence_and_naming.rb

```diff
@@ -52,7 +52,8 @@ def tsort_each_child(node, &block)
   UplinkSet: [:LogicalInterconnectGroup, :LogicalInterconnect],
   Volume: [:StorageSystem, :StoragePool, :VolumeTemplate],
   VolumeAttachment: [:ServerProfile],
-  VolumeTemplate: [:StoragePool]
+  VolumeTemplate: [:StoragePool],
+  PlanScripts: []
```

Comment:
I was in doubt when I was doing this. But I think it's best to separate them. I will do this.